LanguageSkills/Controllers/SubCategoryController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using BusinessLogicLayer;
+using BusinessLogicLayer.Helpers;
+using BusinessLogicLayer.Translation;
+using BusinessLogicLayer.ViewModels;
+using BusinessLogicLayer.ViewModels.Pagination;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LanguageSkills.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SubCategoryController : Controller
+    {
+        private readonly ManageAccessToEntity _manageAccessToEntity = new ManageAccessToEntity();
+
+        /// <summary>
+        /// Get SubCategories with translations by category Id
+        /// </summary>
+        /// <param name="categoryId">Category selected by the user</param>
+        /// <param name="pageNumber">Number of page</param>
+        /// <returns>response status "OK" and list of subCategories with translations or status "NotFound" and error message</returns>
+        [HttpGet("categoryId={categoryId}&pageNumber={pageNumber}")]
```

Comment:
Please make sure that the first param is NOT a main part or URL.

Correct: _URL?categoryId=1&pageNumber=1_
Incorrect (without "?"): 
_URLcategoryId=1&pageNumber=1_ 
or 
_URL/categoryId=1&pageNumber=1_